packages/main/test/specs/Input.spec.js

```diff
@@ -540,8 +536,8 @@ describe("Input general interaction", () => {
 	});
 
 	it("checks if suggestions popover width is maximum 40rem if input isn't wider", async () => {
-		const input = await browser.$("#long-sugg");
-		const listItem = await input.shadow$("ui5-responsive-popover").$("ui5-li-suggestion-item");
+		const input = await $("#input-disabled-autocomplete");
```

Comment:
this change causes the next test that uses "#input-disabled-autocomplete" input to fail (line #684)